### DISCUSSION TITLE
Where can I find recorded Jamulus sessions on my Amazon Lightsail instance ? And how do I turn recording off/on?

### DISCUSSION THREAD

**Discussion by Salpizana** [jamulussoftware/jamulus#2632] [Technical Support & Help] [answered]
I started a server instance on Amazon Lightsail  Ubuntu using the Simon Tomlinson guide. I opted for Record the Session.
Where can I find those recordings?
Thank you.

**Comment by pljones** [comment]
@sthenos can you help?

**Comment by Salpizana** [comment, marked as answer]
I found the answer in Tomlinson's "Idiots Guide to Installing or Upgrading a Jamulus Server on Amazon AWS Lightsail Ubuntu instance" Facebook article at the bottom of the page in the comments section. It is a question several people have asked.

**Edit by @ann0see**: This is the mentioned guide: https://www.facebook.com/notes/jamulus-worldjam/howto-idiots-guide-to-installing-or-upgrading-a-jamulus-server-on-amazon-aws-lig/818091045662521/

**Reply by pljones** [reply]
Could you post it here so anyone finding your question doesn't have to go finding the answer on Facebook?

**Reply by Salpizana** [reply]
https://www.facebook.com/notes/jamulus-worldjam/howto-idiots-guide-to-installing-or-upgrading-a-jamulus-server-on-amazon-aws-lig/818091045662521/

**Reply by jujudusud** [reply]
Isn't it better to go to the official manual page ?
https://jamulus.io/wiki/Server-Linux

**Reply by pljones** [reply]
> Isn't it better to go to the official manual page ? https://jamulus.io/wiki/Server-Linux

Does that provide the answer to where someone following Simon's guide will find their recordings?

**Reply by jujudusud** [reply]
Not directly but it provides the basics to understand what happens when Jamulus records.

**Comment by hsweet** [comment]
I added this line at the end of .bashrc.  Now I have a simple command (record or whatever I call it).  that works in a ssh session terminal.  First time it starts recording, type it again and it stops.
alias record='sudo systemctl kill -s SIGUSR2 servername'

**Reply by Salpizana** [reply]
So I the command would be:
alias record=sudo systemctl kill -s SIGUSR2 anygenre1.jamulus.io:22124

?

**Comment by hsweet** [comment]
The last bit is the name you use for your .service file
/etc/systemctl/system.  for me it is "jamulus_Klezmer.service"  Just leave
out the extension

ExecStart=/usr/bin/jamulus-headless -s -n -p 22125 -u 30 -l
/var/log/jamulus/klezmer.log  -P -T -o "Northeast Klezmer Jam;Montreal;38"
 -R /tmp/jamulusrecording --norecord -e jamulusclassical.fischvolk.de:22524
  -w /usr/local/bin/welcome.html

**Comment by Salpizana** [comment]
Well thankyou so much for replying.
So I tried running this but it failedalias record='sudo systemctl kill -s SIGUSR2 anygenre1.jamulus.io:22124

So as to your new advice, it seems to be completely different 
1) Is this a file I create somewhere(?) and then execute (?). 2) What is  a service file?3) What do you mean by the last bit?3) So what info would you need from me help me set this up?
I know my questions may seem silly to you, but this is all new to meI was a mainframe programmer for years, but BAL and PL/1werethe languages I used.
    On Saturday, May 21, 2022, 09:09:15 AM EDT, Harry Sweet ***@***.***> wrote:  
 
 

The last bit is the name you use for your .service file
/etc/systemctl/system. for me it is "jamulus_Klezmer.service" Just leave
out the extension

ExecStart=/usr/bin/jamulus-headless -s -n -p 22125 -u 30 -l
/var/log/jamulus/klezmer.log -P -T -o "Northeast Klezmer Jam;Montreal;38"
 -R /tmp/jamulusrecording --norecord -e jamulusclassical.fischvolk.de:22524
 -w /usr/local/bin/welcome.html

**Reply by pljones** [reply]
I was a mainframe programmer, too.  Didn't have to do any assembler - just PL/1.  Then someone decided COBOL was the way of the future...  I've been learning new languages ever since.

**Reply by pljones** [reply]
`systemctl kill -s SIGUSR2` sends a signal to a service -- one that's running on your system and managed by the systemd daemon.

You need to give the name of the _service_.  This is NOT the name of the directory your server is registering with, which is what you've written (and not what you were told to write, either, so I don't know where you got it).

Now, if you're not using systemd to manage your Jamulus service, then the advice you've been given isn't of any use to you.  You're better off sticking with a single source -- if you're following Simon's guide, keep using it and ask Simon for help.

There's a completely separate guide on the Jamulus official website, https://jamulus.io/wiki/Server-Linux -- but this relies on you knowing some basics about running a Linux server, from what I remember.

**Comment by Salpizana** [comment]
Good to know there's another mainframer out there. Why there there must be thousands of us!I worked for Eastern Airlines and coded in assembly to make the code faster to execute.So we also used Sabretalk, a sub-dialect of PL/1, and it did add extra code to get where it wanted to go, but what the heck.Doesn't seem to make a difference nowadays

    On Saturday, May 21, 2022, 11:54:04 AM EDT, Peter L Jones ***@***.***> wrote:  
 
 


I was a mainframe programmer, too. Didn't have to do any assembler - just PL/1. Then someone decided COBOL was the way of the future...
